tests/sources/xml/test_marc.py

```diff
@@ -894,11 +968,7 @@ def test_marc_record_missing_leader_logs_error(caplog):
     output_records = Marc("alma", marc_xml_records)
```

Comment:
I'd update this test name since it no longer logs an error

Reply:
I updated the test to say `skips_record` for now, but I think these two tests may be part of the old tests we'll want to deprecate eventually!